### PR TITLE
Add layer wise learning rate notebook

### PR DESCRIPTION
**Pull request by Nic-Ma**
Signed-off-by: Nic Ma <nma@nvidia.com>

This PR added a demo notebook to show how to specify layer wise LR values.

### CODE REVIEW

**Comment by wyli** [comment]
this is nice, a few minor comments:
- perhaps move `print(net)` to the end of the notebook, it's showing a lot of not important information before the important things...
- set `densenet121` `pretrained=True` to start from a pre-trained model, to show that it supports transfer learning
-  we could add this to the highlight document

**Comment by Nic-Ma** [comment]
Hi @wyli ,

Thanks for your review.
I updated the notebook according to your comments.
Could you please help help review it again?
If you think it's OK to merge, could you please help add the link to the highlight page when you adding the GradCam part?

Thanks.

**Review by wyli — APPROVED** [review]
thanks!

**Comment by wyli** [comment]
> Hi @wyli ,
> 
> Thanks for your review.
> I updated the notebook according to your comments.
> Could you please help help review it again?
> If you think it's OK to merge, could you please help add the link to the highlight page when you adding the GradCam part?
> 
> Thanks.

thanks I've added the GradCAM yesterday
